src/main/java/edu/wpi/first/smartdashboard/gui/elements/MjpgStreamerViewerExtension.java

```diff
@@ -103,65 +134,55 @@ public void run() {
             repaint();
           }
 
-        } catch (Exception e) {
+        } catch (IOException ex) {
           imageToDraw = null;
           repaint();
-          e.printStackTrace();
-        }
-
-        if (!ipChanged) {
-          try {
-            Thread.sleep(500);
-          } catch (InterruptedException ex) {
-            // Already fixed in #45
-          }
+          //ex.printStackTrace();
```

Comment:
My mistake.  It was polluting my console because I do not have a camera... Added back in